go/pkg/db/traced_tx.go

```
@@ -0,0 +1,174 @@
+
+// PrepareContext prepares a SQL statement within the transaction with tracing
+func (t *TracedTx) PrepareContext(ctx context.Context, query string) (*sql.Stmt, error) {
+	ctx, span := tracing.Start(ctx, "Tx.PrepareContext")
+	defer span.End()
+	span.SetAttributes(
+		attribute.String("mode", t.mode),
+		attribute.String("query", query),
+	)
+
+	start := time.Now()
+	stmt, err := t.tx.PrepareContext(ctx, query)
```

golangci-lint (2.2.2)

Rows/Stmt/NamedStmt was not closed

(sqlclosecheck)